src/preload/preload.ts

```diff
@@ -39,11 +37,11 @@ export type MainWindowApi = typeof mainWindowApi;
 contextBridge.exposeInMainWorld("mainWindow", mainWindowApi);
 
 const shellApi = {
-    openConfigDir: (): Promise<void> => ipcRenderer.invoke("shell:openConfigDir"),
-    openContentDir: (): Promise<void> => ipcRenderer.invoke("shell:openContentDir"),
-    openSettingsFile: (): Promise<void> => ipcRenderer.invoke("shell:openSettingsFile"),
-    openStartScript: (): Promise<void> => ipcRenderer.invoke("shell:openStartScript"),
-    openReplaysDir: (): Promise<void> => ipcRenderer.invoke("shell:openReplaysDir"),
+    openConfigDir: (): Promise<string> => ipcRenderer.invoke("shell:openConfigDir"),
```

Comment:
These do actually really return strings which I don't think we use. We could update the handle function to a void return type if we wanted to clean this up.